src/utils/stripe/stripeHelpers.tsx

```diff
@@ -28,7 +28,7 @@ export function formatAmountForStripe(
     currencyDisplay: 'symbol',
   });
   const parts = numberFormat.formatToParts(amount);
-  let zeroDecimalCurrency: boolean = true;
+  let zeroDecimalCurrency = true;
```

Comment:
Why did we have to remove the type here ?

Reply:
That was done by `npm run lint:errors -- --fix` as formally regarded as an error by eslint:

```
/Users/norbertschuler/Development/PFTP/planet-webapp/src/utils/stripe/stripeHelpers.tsx
  31:7  error  Type boolean trivially inferred from a boolean literal, remove type annotation  @typescript-eslint/no-inferrable-types

```